esp-hal-common/src/gpio.rs

```diff
@@ -494,6 +494,14 @@ where
                 .modify(|_, w| w.usb_pad_enable().clear_bit());
         }
 
+        // Same workaround as above for ESP32-S3
+        #[cfg(esp32s3)]
```

Comment:
```suggestion
        #[cfg(any(esp32c3, esp32s3))]
```

Too late now that it's merged, but just for future reference it would have been preferable to just update the `cfg` rather than duplicating this entire code block.

Reply:
True - it's the same code

Reply:
Indeed, you are correct. I will create a PR for the corrections later. Thank you for pointing it out.